index.html

```diff
@@ -234,46 +246,25 @@
-                                <td class="form-td"><input type="text" placeholder="Last Name"></td>
-                            </tr>
-                            <tr class="form-tr">
-                                <td colspan="2" class="form-td"><input id="email" type="email" placeholder="E mail address"></td>
-                            </tr>
-                            <tr class="form-tr">
-                                <td colspan="2" class="form-td"><textarea>Typing something</textarea></td>
-                            </tr>
-                            <tr class="form-tr">
-                                <td><button class="see-project-btn">Get in touch</button></td>
-                                <td></td>
-                            </tr>
-                        </table>
+                    <form action="https://formspree.io/f/moqzdnpo" method="post">
+                        <input type="text" name="first-name" placeholder="First Name">
+                        <input type="text" name="last-name" placeholder="Last Name">
+                        <input id="email" name="e-mail" type="email" placeholder="E mail address">
+                        <td colspan="2" name="message" class="form-td"><textarea>Typing something</textarea>
+                        <button class="see-project-btn">Get in touch</button>
                     </form>
                 </div>
                 <div class="touch-class3">
-                    <form>
-                        <table class="form-table">
-                            <tr class="form-tr couple-tr">
-                                <td class="form-td"><input type="text" placeholder="Full Name"></td>
-                            </tr>
-                            <tr class="form-tr">
-                                <td class="form-td"><input id="email" type="email" placeholder="E mail address"></td>
-                            </tr>
-                            <tr class="form-tr">
-                                <td class="form-td"><textarea>Typing something</textarea></td>
-                            </tr>
-                            <tr class="form-tr">
-                                <td align="center"><button class="see-project-btn">Get in touch</button></td>
-                                <td></td>
-                            </tr>
-                        </table>
+                    <form class="mob_form" action="https://formspree.io/f/moqzdnpo" method="post">
+                        <input type="text" name="full-name" maxlength="30" required 
+                         placeholder="Full Name">
+                        <input id="email" name="e-mail" required type="email" placeholder="Email address">
+                        <textarea name="message" maxlength="500" placeholder="Enter text here" required></textarea>
+                        <button class="see-project-btn mob_btn">Get in touch</button>
```

Comment:
- [ ] 😀 From what I noticed here, it seems the submit button has a little `box-shadow` design **on hover** from the template, if this is true, kindly implement it as such 😉.

- Your design
<img width="88" alt="wrong" src="https://user-images.githubusercontent.com/41163571/220489912-5241a717-c264-440a-bf99-1f6ebc67c0a3.png">

- Figma design
<img width="105" alt="right" src="https://user-images.githubusercontent.com/41163571/220489977-cae876c0-a83f-4dd7-9b14-c51d816a2bb0.png">

And Kindly make sure that the color looks similar to that in the design 😀, so as to achieve a pixel-perfect design.


- Figma design

Reply:
![image](https://user-images.githubusercontent.com/54537654/220783395-7304ed31-888e-4207-8696-a5945c34c224.png)
In this version, I've fixed it and now it's properly aligned with figma design.